src/components/index.tsx

```diff
@@ -46,7 +47,7 @@ declare global {
 }
 
 // Configure the redux store.
-const store: Store = configureStore({}, history);
+const store: Store = configureStore({}, history, CPAuth.getInstance());
```

Comment:
Passing the Control Plane authentication instance as an argument, to facilitate testing.